libnestutil/enum_bitfield.h

```diff
@@ -133,10 +133,10 @@ operator^=( Enum& lhs, Enum rhs )
 
 template < typename Enum >
 bool
-enumFlagSet( const Enum en, const Enum flag )
+has_property( const Enum en, const Enum property )
 {
   using underlying = typename std::underlying_type< Enum >::type;
-  return static_cast< underlying >( en & flag ) != 0;
+  return static_cast< underlying >( en & property ) != 0;
 }
```

Comment:
I am not entirely happy with the `has_property()` name here (even though I first suggested it). I think it makes more sense to be able to write something like
```
conn_model.has_property( ConnModProp::IS_PRIMARY )
```
and therefore suggest below to rename `has_flag_set()`. Then having `has_property()` once from here and once from `ConnectorModel` gets confusing. Since the code here is more technical, behind the scenes, maybe calling this `flag_is_set()` and renaming parameters for clarity will give clearer code overall, and fits also better with terminology in the documentation above:
```
template < typename Enum >
bool
flag_is_set( const Enum flags, const Enum flag )
{
   using underlying = typename std::underlying_type< Enum >::type;
   return static_cast< underlying >( flags & flag );
}
```
I also dropped the comparison `!=0`, since anything different from 0 is true (see #2499).

By delegating flag testing then to `ConnectorModel::has_property()`, all code that currently looks like (`has_property` from here)
```
    return has_property( properties_, ConnectionModelProperties::IS_PRIMARY );
```
will become
```
    return has_property( ConnectionModelProperties::IS_PRIMARY );
```
with `has_property()` from `ConnectorModel`.

Sorry I didn't think this properly through in my first review.